tools/bioimaging/bioimage_inference.xml

```diff
@@ -46,16 +52,50 @@
             <param name="input_imaging_model" value="input_imaging_model.zip" location="https://zenodo.org/api/records/6647674/files/weights-torchscript.pt/content"/>
             <param name="input_image_file" value="input_image_file.tif" location="https://zenodo.org/api/records/6647674/files/sample_input_0.tif/content"/>
             <param name="input_image_input_size" value="256,256,1,1"/>
-            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
+            <param name="input_image_input_axes" value="bcyx"/>
+            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="image_diff" />
             <output name="output_predicted_image_matrix" file="output_nucleisegboundarymodel_matrix.npy" compare="sim_size" delta="100" />
         </test>
         <test>
             <param name="input_imaging_model" value="input_imaging_model.zip" location="https://zenodo.org/api/records/6647674/files/weights-torchscript.pt/content"/>
             <param name="input_image_file" value="input_nucleisegboundarymodel.png"/>
             <param name="input_image_input_size" value="256,256,1,1"/>
-            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
+            <param name="input_image_input_axes" value="bcyx"/>
+            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="image_diff" />
```

Comment:
If those images are too large we can use asserts:

https://github.com/BMCV/galaxy-image-analysis/blob/07aba15c8f753d650fee9cdd455896a77086a615/tools/superdsm/superdsm.xml#L171

Reply:
Ok, will try that to check if it works with `TIF` extensions. There are also NPY files which have similar sizes as TIFF files

Reply:
just check there size, but using assert-size